modules/va_facilities/app/controllers/va_facilities/v1/nearby_controller.rb

```diff
@@ -0,0 +1,120 @@
+# frozen_string_literal: true
+
+require 'will_paginate/array'
+
+require_dependency 'va_facilities/application_controller'
+require_dependency 'va_facilities/pagination_headers'
+require_dependency 'va_facilities/geo_serializer'
+require_dependency 'va_facilities/csv_serializer'
+
+module VaFacilities
+  module V1
+    class NearbyController < ApplicationController
+      include ActionController::MimeResponds
+      include VaFacilities::PaginationHeaders
+      skip_before_action(:authenticate)
+      before_action :set_default_format
+      before_action :validate_params, only: [:index]
+
+      REQUIRED_PARAMS = %i[street_address city state zip].freeze
+      TYPE_SERVICE_ERR = 'Filtering by services is not allowed unless a facility type is specified'
+      MISSING_PARAMS_ERR =
+        'Must supply street_address, city, state, and zip to query nearby facilities.'
+
+      def index
+        resource = NearbyFacility.query(params).paginate(page: params[:page], per_page: params[:per_page])
```

Comment:
I think we're pushing past the original intention of facility objects .. the idea is there is base_facility and each type of facility extends that (health, cemetery, etc) .. Nearby will still return facility types, but it may be confusing to consider `nearby` to be a type of facility.. I think the concept of nearby is a controller action construct rather than an object modeling problem

Reply:
Ah, this makes sense. I was going for the expedience of not having to duplicate the maps included in base_facility, but not thinking about how that impacted the conceptual underpinning of the BaseFacility class.

Reply:
Would it make sense to have this inherit from Base Facility in order to avoid duplicating the maps?